src/Node/Render.php

```diff
@@ -85,6 +120,15 @@ public function getComponentParts($component) {
 
   /**
    * Returns the variant context from the config file.
+   *
+   * @param string $variant
+   *   The variant name to get the context for.
+   * @param string $config
+   *   The component config where to lookup the variant in.
```

Comment:
Wouldn't it be more appropriate to pass just the array of variants instead of the whole config?